File: pyg_parent/interface/src/main/java/cn/itcast/core/service/ContentService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.ad.Content;
import cn.itcast.core.pojo.entity.PageResult;

import java.util.List;

public interface ContentService {

    public PageResult search(Integer page, Integer rows, Content content) ;

    public void add(Content content) ;

    public Content  findOne(Long id) ;

    public void update(Content content);

    public void delete(Long[] ids);

    public List<Content> findByCategoryId(Long categoryId);

    public List<Content> findByCategoryIdFromRedis(Long categoryId);
}

File: pyg_parent/web_portal/src/main/webapp/js/service/contentService.js
app.service("contentService",function($http){
	this.findByCategoryId = function(categoryId){
		return $http.get("content/findByCategoryId.do?categoryId="+categoryId);
	}
    this.findItemCatList = function(){
		return $http.get("itemcat/findItemCatList.do?");
	}
});